data-collection/docker-compose.yml

```diff
@@ -58,13 +46,12 @@ services:
       - TASK=worker
       - QUEUE_HOST=queue
       - STORAGE_PATH=/storage
-      - REF_URL=jdbc:mysql://db:3306/refactoringdb?useSSL=false&useLegacyDatetimeCode=false&serverTimezone=UTC
```

Comment:
Same here, let's leave these URLs ready for those who just want to start the docker right away.